pkg/plugins/golang/v3/init.go

```diff
@@ -35,7 +35,7 @@ import (
 
 // Variables and function to check Go version requirements.
 var (
-	goVerMin = golang.MustParse("go1.18.0")
+	goVerMin = golang.MustParse("go1.19.0")
 	goVerMax = golang.MustParse("go2.0alpha1")
```

Comment:
@oscr 

we need to revert this one to be able to move forward, see:

> failed to initialize project: unable to run pre-scaffold tasks of "base.go.kubebuilder.io/v3": go version 'go1.18.6' is incompatible because 'plugin requires go1.19 <= version < go2.0alpha1'. You can skip this check using the --skip-go-version-check flag

Also, based on the changes from 1.18 to 1.19 the solution might still be working with 1.18 so I think we do not need 
to change the min required version for now.

/hold